### DISCUSSION TITLE
Way to purge old audio files?

### DISCUSSION THREAD

**Discussion by mikeschmick** [tphakala/birdnet-go#361] [Q&A]
Hello! 

I have "keep recordings" enabled. I'd like to only keep the current Month, Week, etc. Is this possible? I'm not seeing in wiki or settings options. 

Thank you!

**Comment by tphakala** [comment]
Audio clip retention has two policies, disk utilization threshold and age, of which age policy allows you to define time to keep recordings in days, weeks, months, etc.

![image](https://github.com/user-attachments/assets/d951746a-5f4b-47ce-8bee-94d4b2066b1f)

**Comment by mikeschmick** [comment]
I don't have that option. I pulled newest container update from docker. My side menu looks like this
![Screen Shot 2024-12-31 at 11 00 40 AM](https://github.com/user-attachments/assets/40e7480c-6b0e-48f6-bd48-db3d599a4041)

**Comment by mikeschmick** [comment]
This the correct image to put in my docker compose still?
ghcr.io/tphakala/birdnet-go:latest

**Reply by tphakala** [reply]
pull ghcr.io/tphakala/birdnet-go:dev

**Comment by mikeschmick** [comment]
that did it! Thank you so much! I was missing out on other improvements like the bird pics!
